clients/src/main/resources/common/message/ShareFetchResponse.json

```diff
@@ -54,7 +54,8 @@
           "about": "The acknowledge error code, or 0 if there was no acknowledge error." },
         { "name": "AcknowledgeErrorMessage", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
           "about": "The acknowledge error message, or null if there was no acknowledge error." },
-        { "name": "CurrentLeader", "type": "LeaderIdAndEpoch", "versions": "0+", "fields": [
+        { "name": "CurrentLeader", "type": "LeaderIdAndEpoch", "versions": "0+",
+          "about": "The current leader for this partition.", "fields": [
```

Comment:
In `ShareAcknowledgeResponse` you used `The current leader of the partition.`. Why are we using a different phrasing here?

Reply:
Modified, please review again, thank you!